### DISCUSSION TITLE
[v5] Replace/Remake `/join` command as a `/summon` command

### DISCUSSION THREAD

**Discussion by alou-S** [SudhanPlayz/Discord-MusicBot#698] [General]
#### Description
- Currently the `/join` command just adds the bot to a voice channel which kinda doesn't serve any added purpose since the `/play` command does the same thing anyway.
- Remake the `/join` command  so that if it is used by a **DJ** and is in another channel, the bot will move itself to the channel of the person who called the said command.
- If used by a non privileged user it will join a channel if it isn't in another channel or give a message saying that the user is not privileged enough to move the bot.
- It can be renamed to `/summon` since that word describes the purpose better?

**Comment by jdkenne** [comment]
Hi, how can I assist with completing this issue?
